enrollment-server-onboarding-api-model/pom.xml

```diff
@@ -19,6 +19,11 @@
             <artifactId>enrollment-server-domain-model</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.getlime.core</groupId>
+            <artifactId>rest-model-base</artifactId>
+            <version>1.5.2</version>
```

Comment:
In `mtoken-model` there is 1.5.1? I prefer to omit version here and use maven `dependencyManagement` at parent pom.

Reply:
I will add the version as a configurable property.